contrib/GRAFANA.md

```diff
@@ -0,0 +1,21 @@
+# Grafana README
+
+You can use the provided `grafana.json` dashboard to view metrics for the bridge on Grafana. The dashboard uses the `bridge` label to name bridges
+in a human friendly manner. The prometheus scrape config section below shows how you can add the bridge label to metrics.
```

Comment:
Compound adjective in front of its noun.
```suggestion
in a human-friendly manner. The prometheus scrape config section below shows how you can add the bridge label to metrics.
```

Reply:
ah